### PR TITLE
chore(css): Move CSS examples - Learn CSS, Building Blocks p1

### PR DESCRIPTION
**Pull request by bsmth**
### Description

As part of an initiative to reduce repo maintenance & code duplication, this PR moves some external examples into content. This PR converts the following macros to live samples:

```
{{EmbedGHLiveSample("css-examples/learn/writing-modes/simple-vertical.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/learn/writing-modes/block-inline.html", '100%', 1200)}}
{{EmbedGHLiveSample("css-examples/learn/writing-modes/width.html", '100%', 1200)}}
{{EmbedGHLiveSample("css-examples/learn/writing-modes/inline-size.html", '100%', 1000)}}
{{EmbedGHLiveSample("css-examples/learn/writing-modes/logical-mbp.html", '100%', 1300)}}
{{EmbedGHLiveSample("css-examples/learn/writing-modes/float.html", '100%', 1000)}}
{{EmbedGHLiveSample("css-examples/learn/images/size.html", '100%', 1000)}}
{{EmbedGHLiveSample("css-examples/learn/images/object-fit.html", '100%', 1000)}}
{{EmbedGHLiveSample("css-examples/learn/images/layout.html", '100%', 1000)}}
{{EmbedGHLiveSample("css-examples/learn/images/form.html", '100%', 1000)}}
{{EmbedGHLiveSample("css-examples/learn/overflow/block-overflow.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/learn/overflow/inline-overflow.html", '100%', 600)}}
{{EmbedGHLiveSample("css-examples/learn/overflow/hidden.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/learn/overflow/scroll.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/learn/overflow/scroll-y.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/learn/overflow/scroll-x.html", '100%', 600)}}
{{EmbedGHLiveSample("css-examples/learn/overflow/auto.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/learn/selectors/attribute.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/learn/selectors/attribute-substring.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/learn/selectors/attribute-case.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/learn/selectors/descendant.html", '100%', 500)}}
{{EmbedGHLiveSample("css-examples/learn/selectors/child.html", '100%', 600)}}
{{EmbedGHLiveSample("css-examples/learn/selectors/adjacent.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/learn/selectors/general.html", '100%', 600)}}
{{EmbedGHLiveSample("css-examples/learn/selectors/nesting.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/learn/selectors/selector-list.html", '100%', 1150)}}
```

### Motivation

Motivation is described in the project issue.

### Related issues and pull requests

- [x] https://github.com/mdn/mdn/issues/597

### CODE REVIEW

File: files/en-us/learn/css/building_blocks/handling_different_text_directions/index.md [hunk 1]
```diff
@@ -102,11 +232,61 @@ The {{cssxref("padding-left")}} property maps to {{cssxref("padding-inline-start
...
+```html live-sample___logical-mbp
+<div class="wrapper">
+  <div class="box physical">
+    <h2>Physical Properties</h2>
+    <p>A paragraph. Demonstrating Logical Properties in CSS.</p>
+  </div>
+  <div class="box logical">
+    <h2>Logical Properties</h2>
+    <p>A paragraph. Demonstrating Logical Properties in CSS.</p>
+  </div>
```
Comment (estelle) [inline files/en-us/learn/css/building_blocks/handling_different_text_directions/index.md:248]:
```suggestion
    <p>A paragraph demonstrating logical properties in CSS.</p>
  </div>
  <div class="box logical">
    <h2>Logical Properties</h2>
    <p>A paragraph demonstrating logical properties in CSS.</p>
  </div>
```

File: files/en-us/learn/css/building_blocks/selectors/combinators/index.md [hunk 2]
```diff
@@ -39,15 +39,27 @@ The final selectors we will look at are called combinators, because they combine
 
 ## Descendant combinator
 
-The **descendant combinator** — typically represented by a single space (" ") character — combines two selectors such that elements matched by the second selector are selected if they have an ancestor (parent, parent's parent, parent's parent's parent, etc.) element matching the first selector. Selectors that utilize a descendant combinator are called _descendant selectors_.
+The **descendant combinator** — typically represented by a single space (``) character — combines two selectors such that elements matched by the second selector are selected if they have an ancestor (parent, parent's parent, parent's parent's parent, etc.) element matching the first selector. Selectors that utilize a descendant combinator are called _descendant selectors_.
```
Comment (estelle) [inline files/en-us/learn/css/building_blocks/selectors/combinators/index.md:42]:
```suggestion
The **descendant combinator** — typically represented by a single space (` `) character — combines two selectors such that elements matched by the second selector are selected if they have an ancestor (parent, parent's parent, parent's parent's parent, etc.) element matching the first selector. Selectors that utilize a descendant combinator are called _descendant selectors_.
```

**Review by estelle — APPROVED** [review]
LMostlyGTM!

File: files/en-us/learn/css/building_blocks/selectors/combinators/index.md (hunk 2 above)
Comment (bsmth) [reply files/en-us/learn/css/building_blocks/selectors/combinators/index.md]: Oh, the linter doesn't like space inside code:

```markdown
space (` `) character
```

So I'm going with this:

```markdown
space (<code> </code>) character
```

which seems to be okay

**Comment by bsmth** [comment]
Going to merge now, thank you!

**Pull request merged**